code/script.js

```diff
@@ -225,101 +230,175 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  console.log(secret)
+  }
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
+  winOrLose.style.display = 'none'
+  board.style.display = 'flex' 
   charactersInPlay = CHARACTERS
+  setSecret()
+  generateBoard()
   // What else should happen when we start the game?
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
+  const category = questions.options[questions.selectedIndex].parentNode.label; 
+  const value = questions.options[questions.selectedIndex].value;  
 
+  
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
   // const value =
 
-  currentQuestion = {
-    category: category,
-    // value: value
+
+  if (category === 'vices') {
+    currentQuestion = {
+      attribute: 'vices', 
```

Comment:
Isn't the key "attribute" always the same as the key "category"? I might be out in deep water here, but if that's the case, "attribute" might not be necessary; one could reference the category on the alerts in filterCharacters() instead.